src/main/java/seedu/address/ui/Calender.java

```diff
@@ -0,0 +1,189 @@
+            }
+        } else {
+            // do nothing
+        }
+        monthYearInput.setText(this.headerMonth.toString() + " " + this.headerYear);
+        monthYearInput.setStyle("-fx-text-fill: white");
+    }
+
+    private void loadNow() {
+        emptyCal();
+        loadDayHeader();
+        updateHeader(0);
+        this.month.firstDayOfYear(this.isLeapYear);
+        LocalDate start = YearMonth.of(this.headerYear, this.headerMonth).atDay(1);
+        int first = start.getDayOfWeek().getValue();
+        int count = 1;
+        for (int i = first; i < first + numOfDays(); i++) {
+            Label label = new Label();
+            label.setText(Integer.toString(count));
+            label.setStyle("-fx-text-fill: white");
+            count++;
+            VBox box = new VBox();
+            box.setAlignment(Pos.CENTER);
+            box.getChildren().add(label);
+            if (count % 2 == 0) {
+                box.setBackground(new Background(
+                        new BackgroundFill(Color.DIMGRAY, CornerRadii.EMPTY, Insets.EMPTY)));
+            }
+            box.setBorder(new Border(new BorderStroke(Color.valueOf("white"),
+                    BorderStrokeStyle.SOLID, CornerRadii.EMPTY, new BorderWidths(1))));
+            int col = parseCol(i);
+            int row = parseRow(i);
+            fillCalender(box, col, row);
+        }
+    }
+
+    private int parseCol(int value) {
+        return (value + 6) % 7;
+    }
+
+    private int parseRow(int value) {
+        return (value - 1) / 7 + 1;
+    }
+
+    private void emptyCal() {
+        calenderGrid.getChildren().clear();
+    }
+
+    private void loadDayHeader() {
+        for (int i = 0; i < 7; i++) {
+            switch (i) {
+            case 0:
+                setDayHeader("Monday", 0);
+                break;
+            case 1:
+                setDayHeader("Tuesday", 1);
+                break;
+            case 2:
+                setDayHeader("Wednesday", 2);
+                break;
+            case 3:
+                setDayHeader("Thursday", 3);
+                break;
+            case 4:
+                setDayHeader("Friday", 4);
+                break;
+            case 5:
+                setDayHeader("Saturday", 5);
+                break;
+            case 6:
+                setDayHeader("Sunday", 6);
+                break;
+            default:
+                setDayHeader("", 0);
+            }
+        }
+    }
+
+    private void setDayHeader(String day, int col) {
+        VBox box = new VBox();
+        Label label = new Label();
+        label.setText(day);
+        label.setUnderline(true);
+        label.setStyle("-fx-text-fill: white");
+        box.setAlignment(Pos.CENTER);
+        box.getChildren().add(label);
+        fillCalender(box, col, 0);
+    }
+
+    /**
+     * Handles the button action when pressing on next month.
+     */
+    @FXML
+    public void nextMonth() {
+        updateHeader(1);
```

Comment:
Same issue as above. Maybe we can name it `handleNextMonthButton()`? It is much longer though, but in return it is much more intuitive and also enhance the readability.